src/analyses/callstringTracking.ml

```diff
@@ -0,0 +1,36 @@
+open Analyses
+
+module Spec : MCPSpec =
+struct
+  include Analyses.IdentitySpec
+
+  let name () = "callstringTracking"
+
+  module FundecSet = SetDomain.Make (CilType.Fundec)
+  module FundecList = Lattice.Flat(Printable.Liszt(CilType.Fundec))
+  module D = Lattice.Prod (FundecList) (FundecSet)
+  module C = D
+  module G = D
+  module V = EmptyV
+  let startstate v = D.bot ()
+  let exitstate  v = D.bot ()
+
+  let get_list l = match l with
+    | `Lifted l -> l
+    | `Bot -> []
+    | `Top -> failwith "callstringTracking Error: the value of the Flat List shouldn't be top!"
+
+  let callee_state f ctx = 
+    let (list, set) = ctx.local in
```

Comment:
`list` as a name is a bit suboptimal, it confuses the syntax highlighting.

Reply:
is `stack `ok?

Reply:
Sure, it's not a keyword.